.env.example

```
@@ -0,0 +1,122 @@
+
+# ===== LLM Configuration =====
+# Add your OpenAI API key for LLM-dependent features (image alt generation, etc.)
+# Get your key from: https://platform.openai.com/api-keys
+OPENAI_API_KEY=your-openai-api-key-here
+MODEL_NAME=gpt-4o
+
+# ===== Vector Search & Embeddings (TEI) =====
+# Text Embeddings Inference service configuration
+TEI_URL=http://localhost:8080
+TEI_API_KEY=
+ENABLE_VECTOR_STORAGE=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENABLE_VECTOR_STORAGE key should go before the TEI_API_KEY key

(UnorderedKey)